src/util/deprecated.js

```diff
@@ -0,0 +1,5 @@
+export default function deprecated (elem, oldUsage, newUsage) {
+  // TODO: add logic to only warn in dev mode
```

Comment:
Happy to either leave this as is and raise an issue to address it, or to do it now. If nether then I'd rather remove it as it'd be unused. I'll leave it up to you.